scylla/src/transport/connection.rs

```diff
@@ -914,7 +914,7 @@ impl Connection {
             prepared_queries.insert(query, prepared);
         }
 
-        let mut batch: Cow<Batch> = Cow::Owned(Default::default());
+        let mut batch: Cow<Batch> = Cow::Owned(Batch::new(init_batch.get_type()));
```

Comment:
Are we sure that there are no more config values that need to be copied?
Maybe a bit cleaner solution would to to introduce `new_from` method on Batch?